### PR TITLE
Setup BookStore project

### PR DESCRIPTION
**Pull request by jheart-vic**
Hi,
   - I created a navigation bar for the app
   - I added  two `<Route>`s and `<Link>`s for the app's navigation:
   - Books - the default view:
   -  It displays the list of books (empty at this point but it should be ready for the data) with the Remove button.
   - I created a component called Book for displaying a single book and reused it in a component that displays a list of books.
   - I have a form for adding a book 
   - Categories:
   - It displays a button "Check status" only.
   - kindly review
   Thanks

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+//       <div className="content3">
+//         <span className="currentChapter">CURRENT CHAPTER</span>
+//         <span className="chapter">Chapter 17</span>
+//         <button type="button" className="updateBtn">UPDATE PROGRESS</button>
+//       </div>
+
+//     </div>
+//   );
+// };
```
Comment (rustam-tolipov) [inline src/components/Books.js]: - [ ] Kindly remove unused commented codes from your project.

File: src/components/NavBar.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  height: 3rem;
+  margin: 0 0 2rem;
+  padding: 1.4rem 4rem 1.7rem 4.3rem;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid #121212;
+  align-items: center;
+
+  /* box-shadow: 0 4px 20px -3px rgba(0, 0, 0, 0.71); */
```
Comment (rustam-tolipov) [inline src/components/NavBar.css]: - [ ] If you don't need commented lines of codes please remove it from your project.

File: src/components/BookContainer.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import './Book.css';
+
+const BookContainer = ({ books }) => (
+  <Router>
+
+    <div className="div">
+      <NavBar />
+      <Routes>
+        <Route path="/" exact element={<Books books={books} />} />
+        <Route path="/category" element={<Category />} />
```
Comment (rustam-tolipov) [inline src/components/BookContainer.js:16]: - [ ] Your `add new book` section rendering in Books and Categories. Why didn't you put your `add new book` part into Books page. In requirement, it only appears in Books page, not in both pages. Kindly follow the requirement.

File: README.md [hunk 4]
```diff
@@ -38,7 +38,8 @@ Template from this [MICROVERSE](https://www.microverse.org/)
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/jheart-vic/BookStore.git/issues).
+Feel free to check the [issues page](https://github.com/jheart-vic/BookStore.git
+/issues).
 
 ## Contributers
```
Comment (rustam-tolipov) [inline README.md]: - [ ] [OPTIONAL] It seems like you forgot to add MIT license to your project. Kindly implement MIT license in your project as well.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @jheart-vic, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi @jheart-vic 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**